src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double basis1;
+    private double basis2;
```

Comment:
don't use numbers in naming, it's usually a bad practice
let's call it topBase, bottomBase or smth like this